db/storage.go

```diff
@@ -0,0 +1,148 @@
+package db
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/google/uuid"
+)
+
+type StorageLog struct {
+	DealUUID  uuid.UUID
+	CreatedAt time.Time
+	PieceSize abi.PaddedPieceSize
+	Text      string
+}
+
+type StorageDB struct {
+	db *sql.DB
+}
+
+func NewStorageDB(db *sql.DB) *StorageDB {
+	return &StorageDB{db: db}
+}
+
+func (s *StorageDB) Tag(ctx context.Context, dealUuid uuid.UUID, pieceSize abi.PaddedPieceSize) error {
```

Comment:
Would it make sense to use uint64 instead of abi.PaddedPieceSize at the DB layer?

Reply:
Yes, actually it is simpler.